geoapps/inversion/electricals/direct_current/three_dimensions/__init__.py

```diff
@@ -0,0 +1,10 @@
+#  Copyright (c) 2022 Mira Geoscience Ltd.
```

Comment:
Does the linter complain if we call this 3D?

Reply:
you bet!  open to suggestions